microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/compatibility/MissingJwtTest.java

```diff
@@ -0,0 +1,101 @@
+import org.jboss.eap.qe.microprofile.jwt.EnableJwtSubsystemSetupTask;
+import org.jboss.eap.qe.microprofile.jwt.security.rbac.jwe.JweTestCase;
+import org.jboss.eap.qe.microprofile.jwt.testapp.Endpoints;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.JaxRsTestApplication;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.SecuredJaxRsEndpoint;
+import org.jboss.eap.qe.microprofile.jwt.testapp.servlet.SecuredServlet;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * The @see org.eclipse.microprofile.auth.LoginConfig annotation provides the same information as the web.xml login-config
+ * element; this tests verifies that, when present, the following behaviours are the same for both `MP-JWT` and `BASIC`
+ * authentication:
+ * <ul>
+ * <li>no authentication provided, be it JWT token or BASIC, leads to HTTP 200</li>
+ * <li>wrong authentication provided, be it JWT token or BASIC, leads to HTTP 401</li>
+ * </ul>
+ */
+@RunAsClient
+@RunWith(Arquillian.class)
+@ServerSetup(EnableJwtSubsystemSetupTask.class)
+public class MissingJwtTest {
+
+    private static final String MP_JWT_AUTHENTICATION_DEPLOYMENT = "MissingJwtTest-first-deployment";
+    private static final String BASIC_AUTHENTICATION_DEPLOYMENT = "MissingJwtTest-second-deployment";
+
+    @Deployment(name = MP_JWT_AUTHENTICATION_DEPLOYMENT)
+    public static WebArchive createDeploymentMpJwtAuth() {
+        return ShrinkWrap.create(WebArchive.class, MP_JWT_AUTHENTICATION_DEPLOYMENT + ".war")
+                .addClass(SecuredJaxRsEndpoint.class)
+                .addClass(JaxRsTestApplication.class)
+                .addAsManifestResource(MissingJwtTest.class.getClassLoader().getResource("mp-config-basic-RS256.properties"),
+                        "microprofile-config.properties")
+                .addAsManifestResource(MissingJwtTest.class.getClassLoader().getResource("pki/RS256/key.public.pem"),
+                        "key.public.pem");
+    }
+
+    @Deployment(name = BASIC_AUTHENTICATION_DEPLOYMENT)
+    public static WebArchive createDeploymentBasiAuth() {
+        return ShrinkWrap.create(WebArchive.class, BASIC_AUTHENTICATION_DEPLOYMENT + ".war")
+                .setWebXML(JweTestCase.class.getClassLoader().getResource("basic-authentication.web.xml"))
+                .addClass(SecuredServlet.class);
+    }
+
+    /**
+     * @tpTestDetails Supply an expired ({@code exp} set in past) JWT to server
+     * @tpPassCrit Expired JWT is rejected and user receives 401/Unauthorized.
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    public void supplyNoJwtToServerTest(@ArquillianResource @OperateOnDeployment(MP_JWT_AUTHENTICATION_DEPLOYMENT) URL url) {
+        given()
+                .when().get(url.toExternalForm() + Endpoints.SECURED_ENDPOINT + "/protected")
+                .then()
+                .statusCode(200);
+    }
+
+    @Test
```

Comment:
Please document the test cases https://github.com/jboss-eap-qe/eap-microprofile-test-suite/blob/master/CONTRIBUTING.md#test-documentation

Reply:
Still missing